KlarnaOnDemand/KODRegistrationResult.h

```diff
@@ -10,6 +10,16 @@
  */
 @property (strong, nonatomic) NSString *token;
 
-- (id)initWithToken:(NSString *)token;
+/**
+ *  The user's validated phone number.
+ */
+@property (strong, nonatomic) NSString *phoneNumber;
+
+/**
+ *  The user's registration details.
```

Comment:
This is really too vague. At the very least we need some link to some docs that don't exist. Talk to the Goor to iron this out, won't you?